webapp/lib/view.dart

```diff
@@ -44,16 +45,19 @@ void init() {
   document.onKeyDown.listen((event) => command(UIAction.keyPressed, new KeyPressData(event.key)));
 }
 
-void initSignedInView() {
+void initSignedInView() async {
```

Comment:
We need to be a bit careful with turning methods into an async ones, and I'd rather us not do that here. If for some reason Nook can't read the commit hash file, it will crash, and the UI won't load at all. Reading the hash is more for debugging than core functionality - Nook should be able to run without it.

Can you instead leave the signed in status as it was, without the hash, and afterwards attempt to read the file and replace `showNormalStatus` with the commit hash?

Reply:
Refactored this to get hash afterwards